spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -591,8 +591,6 @@ For each field of the metamodel class that is to be initialized by a Jakarta Dat
 - The name of the field, ignoring case, must match the name of an entity attribute, with the `_` character in the field name delimiting the attribute names of hierarchical structures or relationships, such as embedded classes.
 - The value of the field must be uninitialized or `null`.
 
-If the entity class has a unique identifier that is represented by a single entity attribute, then the application can also include a field that follows the above criteria, except that it has a field name of `id` or `ID`.
```

Comment:
This is one of the places. I believe it also needs to be removed from StaticMetamodel JavaDoc:

```
 * <p>Additionally, a field that meets the above criteria except for the name and is named
 * {@code id} or {@code ID} must be assigned by the Jakarta Data provider for the unique
 * identifier entity attribute if a single entity attribute represents the unique identifier.</p>
```

Reply:
Right, missed that one. Thanks, fixed.